tests/unit/utils.py

```diff
@@ -11,6 +11,7 @@
 from hdmf.utils import (docval, getargs, get_docval)
 
 CORE_NAMESPACE = 'test_core'
+ROOT_NAME = 'root'
```

Comment:
Is there a reason why we would not just import it?

```suggestion
from hdmf_zarr.backend import ROOT_NAME
```

Reply:
I don't think so. I think the manual definition within utils.py is a legacy carry over from HDMF, where it is set with all the Foo and Baz containers and not imported.